test/trento/application/projectors/check_result_projector_test.exs

```diff
@@ -2,6 +2,9 @@ defmodule Trento.CheckResultProjectorTest do
   use ExUnit.Case
   use Trento.DataCase
 
+  import Phoenix.ChannelTest
+  import TrentoWeb.ChannelCase
```

Comment:
So at the end the trick was to import this instead of `use`ing.
That's why you need to import the `Phoenix.ChannelTest`. 
Most probably you cannot `use` both because there is a conflict starting the sandbox

Reply:
exactly, only one type of test case should be "used" because then the sandbox init will conflict on both